data/projections.csv

```diff
@@ -5,11 +5,11 @@ R,J,M,1,2,3 osa bis co;obersubalpin;99999;99999;99999;99999;;;;;;subalpin
 R,J,M,1,2,3 osa bis co;obersubalpin;59;< als 60% ;Boden normal;57V;;;;;;subalpin
 R,J,M,1,2,3 osa bis co;obersubalpin;59;< als 60% ;Boden normal;57V;;;;;;subalpin
 R,J,M,1,2,3 osa bis co;obersubalpin;99999;99999;99999;99999;;;;;;subalpin
-R,J,M,1,2,3 osa bis co;obersubalpin;59Lä;> als 60% ;Boden normal;57CLä;;;;;;subalpin
-R,J,M,1,2,3 osa bis co;obersubalpin;59Lä;> als 60% ;Boden normal;57CLä;;;;;;subalpin
+R,J,M,1,2,3 osa bis co;obersubalpin;59LÃĊ;> als 60% ;Boden normal;57CLÃĊ;;;;;;subalpin

```

Comment:
Seems like this introduce a encoding bug. Can you please check why this changes?

Reply:
Found the reason, seems like it happened during the encoding to UTF-8. The original file is safe so will fix it soon.